src/decorators/query-handler.decorator.ts

```diff
@@ -2,6 +2,10 @@ import 'reflect-metadata';
 import { IQuery } from '../interfaces';
 import { QUERY_HANDLER_METADATA } from './constants';
 
+/**
+ * QueryHandler's are responsive for the queries dispatch in `QueryBus` and usually used to `reads`
+ * @see https://docs.nestjs.com/recipes/cqrs#queries
```

Comment:
```suggestion
 * @param query `IQuery `
 * @see https://docs.nestjs.com/recipes/cqrs#queries
```

Reply:
Thanks for the suggestion, I kinda forgot at some point of adding the params